client/src/pages/Signup/SignupParents/parentSignupService.ts

```diff
@@ -5,10 +5,10 @@ interface IAxiosResponse {
   message?: string;
 }
 
-export const parentSignup = async (data: FieldValues) => {
+export const parentSignup = async (data: FieldValues): Promise<void> => {
   try {
     const { termsAndConditions, confirmPassword, ...parentDetails } = data;
-    return await axios.post(
+    await axios.post(
```

Comment:
you just need to write return axios.post();
await - is not needed cause you call this function with await
and return - is needed so this will work